package.json

```diff
@@ -9,7 +9,7 @@
   "author": "Brainly - Konrad Dzwinel <konrad.dzwinel@brainly.com>",
   "license": "MIT",
   "scripts": {
-    "test": "eslint html2asketch/ asketch2sketch/ && jest",
+    "test": "jest",
```

Comment:
we definitely want to keep eslint ;)

Reply:
it was just annoying to run eslint for tests all the time.

Reply:
do you mind to have validate script to combine eslint and test to run in prepush hook?

Reply:
"test" should run both - linting and unit testing, but we sure can have separate "lint" and "unit-test" commands for running them separately if that's useful for you. We can also run "test" in the prepush hook 👍